realm/realm-library/src/main/java/io/realm/internal/Table.java

```diff
@@ -1237,7 +1237,7 @@ public long findFirstDate(long columnIndex, Date date) {
         if (date == null) {
             throw new IllegalArgumentException("null is not supported");
         }
-        return nativeFindFirstDate(nativePtr, columnIndex, date.getTime() / 1000);
+        return nativeFindFirstTimestamp(nativePtr, columnIndex, date.getTime() / 1000);
```

Comment:
We have a test but I disabled it initially and apparently forgot to reenable it. Nice catch!